modules/rhel-compute-updating.adoc

```diff
@@ -41,9 +41,9 @@ You must not enable firewalld later. If you do, you cannot access {product-title
 +
 ----
 # subscription-manager repos --disable=rhel-7-server-ansible-2.7-rpms  \
```

Comment:
@mtnbikenc, for a 4.4 to 4.5 upgrade, should we remove the step to swap the Ansible repo?

Reply:
It looks like we missed the requirement to upgrade to Ansible 2.9 for 4.4, so we should ensure 2.9 is enabled for 4.5.

4.3 requires [Ansible 2.8](https://github.com/openshift/openshift-ansible/blob/release-4.3/openshift-ansible.spec#L20)
4.4 requires [Ansible 2.9](https://github.com/openshift/openshift-ansible/blob/release-4.4/openshift-ansible.spec#L20)
4.5 requires [Ansible 2.9](https://github.com/openshift/openshift-ansible/blob/release-4.5/openshift-ansible.spec#L20)